.github/workflows/deploy_tre_reusable.yml

```diff
@@ -59,17 +72,25 @@ on:
 
 # This will prevent multiple runs of this entire workflow.
 # We should NOT cancel in progress runs as that can destabilize the environment.
-concurrency: "deploy-${{ github.ref }}"
+concurrency: "deploy-${{ inputs.ciGitRef }}"
 
 jobs:
   deploy_management:
     name: Deploy Management
     runs-on: ubuntu-latest
     environment: Dev
     steps:
-      - name: Checkout
+      - name: Checkout main
+        if: ${{ inputs.prRepo == '' }} # if not running for a PR, checkout main
```

Comment:
This won't always checkout main, for example if deploy_tre is "dispatched" (I'm doing it myself sometimes). So maybe "default" is more appropriate here.

Reply:
Ah yes - I have updated the name & comment